content/pages/shop.md

```diff
@@ -12,5 +12,4 @@ meta:
   title: Shop
 ---
 
-
```

Comment:
Avoid all of these white space changes in commits, make a seperate style commit if you need to.